ui/src/Explore/Components/ContentsView.test.tsx

```diff
@@ -0,0 +1,140 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import ContentView from "./ContentView";
+// tslint:disable-next-line:no-implicit-dependencies
+import nock from "nock";
+import { Content, Dialect } from "../../utils";
+
+it("Render Iceberg Contents", async () => {
+  const scope = nock("http://localhost/api/v1")
+    .get(
+      "/contents?hashOnRef=55cc2e40cbe2eed190f3987c53ffc34643d5cfa5ef878392d88b2eb4137e92e4&ref=dev"
+    )
+    .reply(200, { contentsList });
+  const { getByAltText, asFragment } = render(
+    <BrowserRouter>
+      <ContentView tableContents={iceBergContent} />
+    </BrowserRouter>
+  );
+  expect(asFragment()).toMatchSnapshot();
+  await waitFor(() => getByAltText("IcebergTable"));
+  setTimeout(() => {
+    scope.done();
+  }, 5000);
+  expect(asFragment()).toMatchSnapshot();
+  expect(screen.getByAltText("IcebergTable")).toHaveAttribute(
+    "src",
+    "/iceberg.png"
+  );
+});
+
+it("Render DeltaLake Contents", async () => {
+  const scope = nock("http://localhost/api/v1")
+    .get(
+      "/contents?hashOnRef=55cc2e40cbe2eed190f3987c53ffc34643d5cfa5ef878392d88b2eb4137e92e4&ref=dev"
+    )
+    .reply(200, { contentsList });
+  const { getByAltText, asFragment } = render(
+    <BrowserRouter>
+      <ContentView tableContents={detlaLakeContent} />
+    </BrowserRouter>
+  );
+  expect(asFragment()).toMatchSnapshot();
+  await waitFor(() => getByAltText("IcebergTable"));
+  setTimeout(() => {
+    scope.done();
+  }, 5000);
+  expect(asFragment()).toMatchSnapshot();
+  expect(screen.getByAltText("IcebergTable")).toHaveAttribute(
+    "src",
+    "/delta.png"
+  );
+});
+
+it("Render SQL View Contents", async () => {
+  const scope = nock("http://localhost/api/v1")
+    .get(
+      "/contents?hashOnRef=55cc2e40cbe2eed190f3987c53ffc34643d5cfa5ef878392d88b2eb4137e92e4&ref=dev"
+    )
+    .reply(200, { contentsList });
+  const { getByText, asFragment } = render(
+    <BrowserRouter>
+      <ContentView tableContents={sqlViewContent} />
+    </BrowserRouter>
+  );
+  expect(asFragment()).toMatchSnapshot();
+  await waitFor(() => getByText("VIEW"));
+  setTimeout(() => {
+    scope.done();
+  }, 5000);
+  expect(asFragment()).toMatchSnapshot();
+  expect(screen.getByText("VIEW")).toBeInTheDocument();
+});
+
+const iceBergContent: Content = {
+  type: "ICEBERG_TABLE",
+  id: "5d8bdfec-96d7-44f6-8770-1ffcf09e044d",
+  metadataLocation: "/path/to/metadata/",
+  idGenerators: "6c3563f6847feda9",
```

Comment:
Also as per new release `idGenerators` is removed. 

please use the latest ICEBERG_TABLE content.

Reply:
https://github.com/projectnessie/nessie/pull/2626

Reply:
@ajantha-bhat Looks like the following fields are newly added, do we have to show them on UI? it seems like those are internal purposes only but wanted to make sure.
 schemaId, specId, sortOrderId
 cc: @nastra

Reply:
let's omit them for now. We can always add them later to the UI if necessary

Reply:
@ajantha-bhat idGenerators is removed from test cases. Also updated latest content for same.